frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.scss

```diff
@@ -22,3 +22,7 @@
 .nb-object-data-reduction-card__row-title {
   margin-right: 0.5em;
 }
+
```

Comment:
Can we get rid of line 1? I don't think it's being used anywhere.

Reply:
Sure.